### PR TITLE
Fix YouTube playback in multimedia player on iOS

### PR DESCRIPTION
**Pull request by jeffdavidgordon**
This is a solution for issue #7521.   It adds a bit of code to the multimedia player so that when a YouTube video on iOS is detected, the multimedia player custom controls are not loaded and defaults to YouTube controls.

### CODE REVIEW

**Comment by LaurentGoderre** [comment]
This would make the player not be accessible anymore. We shouldn't use this workaround

**Comment by jeffdavidgordon** [comment]
This workaround is only for iPhones/iPad.   Right now, iPhones/iPads don't play videos at all!    Note that this workaround is not used for any other player / platform combination.

File: src/plugins/multimedia/multimedia.js [hunk 1]
```diff
@@ -628,11 +638,20 @@ $document.on( youtubeEvent, selector, function( event, data ) {
 			$this = $( event.currentTarget ),
 			$media, ytPlayer;
 
+		// if YouTube on iOS, load YouTube's controls.  Due to restrictions by Apple
+		// and Youtube, videos cannot be played with external controls ( i.e. playVideo() )
+		var $iOS = ( /iPad|iPhone|iPod/.test( navigator.userAgent ) && !window.MSStream );
+		$iOS = true;
```
Comment (nschonni) [inline src/plugins/multimedia/multimedia.js]: This means the accessible controls will never load

Comment (jeffdavidgordon) [inline src/plugins/multimedia/multimedia.js]: There's an error in the code that needs to be fixed.   The $ios=True line should be removed, it was for testing purposes only.   Will fix.

**Comment by jeffdavidgordon** [comment]
PR has now been fixed, test code has been removed.

**Comment by jeffdavidgordon** [comment]
@LaurentGoderre @nschonni Are you able to accept this PR?   Are there any outstanding issues?  Thanks!

**Comment by hsrudnicki** [comment]
Can be this pull closed due to the pass results/checks?

**Comment by jeffdavidgordon** [comment]
Can you merge this pull into master?

**Comment by nschonni** [comment]
Yes, this fixes an issue, but it would make the player inaccessible, that's why @LaurentGoderre or I didn't merge it. If one of the current maintainers wants to accept that, it's up to them.

**Comment by jeffdavidgordon** [comment]
Can you confirm how it makes the player inaccessible?   There was originally an erroneous line that caused the fix to invoke in all cases, but that was removed.

**Comment by masterbee** [comment]
@LaurentGoderre @nschonni do you mind clarify what makes it inaccessible? Is it just that the fix offloads to YouTube controls for those devices which we already know has accessibility issues in its controls, or is this something more?

**Comment by LaurentGoderre** [comment]
@masterbee YouTube's controls are not accessible. Is it still an issue since we removed the overlay for YouTube players? Can we confirm?

**Comment by masterbee** [comment]
@jeffdavidgordon - can you test with the latest build to see if this is still an issue? The overlay as @LaurentGoderre mentioned was problematic for mobile families at times.

**Comment by shawnthompson** [comment]
Working example of PR
https://shawnthompson.github.io/jeffdavidgordon-master/unmin/demos/multimedia/multimedia-youtube-en.html
https://github.com/shawnthompson/jeffdavidgordon-master

**Comment by LaurentGoderre** [comment]
This should not be merged. A more robust solution needs to be found

**Comment by lucas-hay** [comment]
Discussed and agreed in PR review meeting to close this PR and not merge. This introduces accessibility issues, and is simply a "bandaid" to the real issue (#7521)

**Pull request closed without merging**